### PR TITLE
Add GPO Doc for server address

### PR DESCRIPTION
**Pull request by JtheBAB**
This pull request add the gpo doc for server addresses and prevent server. New also with type of GPO.

### CODE REVIEW

**Comment by JtheBAB** [comment]
Perhaps Computer is better then Machine?

File: source/install/desktop.rst [hunk 1]
```diff
@@ -43,11 +43,15 @@ then open the Group Policy Management Console and you should see under "Computer
...
 
-+------------------------+-----------------------------------------------------+----------------------+
-| Group Policy           | Description                                         | Required Version     |
-+========================+=====================================================+======================+
-| Disable Auto Updater   | Controls the behavior of the internal Autoupdater   | 4.3 or later         |
-+------------------------+-----------------------------------------------------+----------------------+
++--------------------------------------+---------------------------------------------------+----------------+----------------------+
+| Group Policy                         | Description                                       | Type of Policy | Required Version     |
++======================================+===================================================+================+======================+
+| Disable Auto Updater                 | Controls the behavior of the internal Autoupdater | Machine        | 4.3 or later         |
```
Comment (amyblais) [inline source/install/desktop.rst]: Please be consistent with the spelling of "Auto updater"

Comment (JtheBAB) [reply source/install/desktop.rst]: done :)

**Comment by wget** [comment]
Don't merge this PR until we are not ready with msi.

We might standardize the installer `argument` with the ones of GPO in order to get some consistency.

**Comment by amyblais** [comment]
@JtheBAB Dean prepared GPO docs here: https://github.com/mattermost/docs/pull/2668.

Wondering if this doc PR is still relevant or can be closed? If still relevant, please ignore this comment and we can keep this open.

**Comment by JtheBAB** [comment]
No longer needed :)

**Pull request closed without merging**